examples/crds/ciliumnetworkpolicies.yaml

```diff
@@ -31,6 +31,9 @@ spec:
           apiVersion:
             description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
             type: string
+          description:
+            description: Description is a human-readable description of the resource.
+            type: string
```

Comment:
I thought the idea was not allowing this in v1.

Reply:
Yes, will remove.